File: CursoPOOUber/Csharp/UberX.cs
using CarNS;
using AccountNS;

namespace CarTypes.UberXCar
{
    //Definition of the "UberX" class, which inherits from the "Car" class.
    public class UberX : Car
    {
        
        //Attributes
        public string Brand { get; set; }
        public string Model { get; set; }

        public UberX(string License, Account Driver, string brand, string model) : base(License, Driver)
        {
            this.Brand = brand;
            this.Model = model;
            SetPassengers(4);
        }

        public override void PrintDataCar()
        {
            base.PrintDataCar();
            Console.WriteLine("Car Brand: " + Brand + " UberX model: " + Model);
        }
    }
}

File: CursoPOOUber/PHP/uberX.php
<?php
require_once('Car.php');
class UberX extends Car {
    public $brand;
    public $model;


    public function __construct($license, $driver, $brand, $model){
        parent::__construct($license,$driver);
        $this->brand = $brand;
        $this->model = $model;
    }

    public function setPassenger($passenger) {
        if ($passenger == 4) {
            $this->passenger = $passenger;
        } else {
            echo "This type of Uber is only for 4 passengers.";
        }
    }
    
    public function printDataCar() {
        echo "Car brand: $this->brand; <br>
                UberX model: {$this->model} <br>";
    }
}
?>